ai-ml/trainium-inferentia/variables.tf

```diff
@@ -140,3 +140,9 @@ variable "kms_key_admin_roles" {
   type        = list(string)
   default     = []
 }
+
+variable "enable_rayserve_ha_elastic_cache_redis" {
+  description = "Flag to enable Ray Head High Availability with Elastic Cache for Redis"
+  type        = bool
+  default     = true
```

Comment:
Let's make this `false` as default and enable it as a part of the RayHA deployment. Just to avoid deploying this cluster for other training blueprints